### PR TITLE
improvement(perf during ops): change operations order

### PR DESCRIPTION
**Pull request by fgelcer**
as we are suffering in mixed workload from really
bad replace node results, we decided that we will
run this operation when the cluster will be larger, reducing the overhead it causes on a 3 nodes cluster, but now doing it on a 6 nodes cluster (between operations `add_node` and `decommission`, so we should get better results for replace node, and not affect any other operations we have currently.

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Comment by fgelcer** [comment]
giving it a ride [in here](https://jenkins.scylladb.com/job/scylla-staging/job/fabio/job/perf-acceptance-criteria/job/perf-regression-latency-650gb-with-nemesis/3/)

**Comment by fgelcer** [comment]
> giving it a ride [in here](https://jenkins.scylladb.com/job/scylla-staging/job/fabio/job/perf-acceptance-criteria/job/perf-regression-latency-650gb-with-nemesis/3/)

right now it is ending the terminate and replace step... it will now "shrink" the cluster, and it may take another few hours until ending... and then we can check the results

```
11:14:56  < t:2023-07-31 08:14:54,933 f:file_logger.py  l:101  c:sdcm.sct_events.file_logger p:INFO  > 2023-07-31 08:14:54.485: (InfoEvent Severity.NORMAL) period_type=not-set event_id=7621d0aa-fef6-447f-9a8f-2c2f7c3117d1: message=Finished terminate_and_replace disruption
```

**Comment by fgelcer** [comment]
result email was sent to our QA perf team email group
`test_id: d8d56fad-84a0-48f5-8055-ea8357f612b2`

it did improve the results of replace node operation

**Review by roydahan — APPROVED** [review]
Approved, but run it through the performance meeting before merging and backporting.

**Comment by fgelcer** [comment]
@aleksbykov @temichus , if you approve it too, then if Avi / Yaniv approve such a change as well, we can merge and backport, so the weekend run will use this new format

**Review by aleksbykov — APPROVED** [review]

**Comment by aleksbykov** [comment]
@fgelcer , i talked with Avi, he approved the order. So we can merge PR

**Comment by fgelcer** [comment]
@scylladb/qa-maintainers , can you please merged and backport?

**Pull request merged**
